pepper-apis/docs/specification/src/pepper.yml

```diff
@@ -102,6 +102,8 @@ paths:
     $ref: 'endpoints/user.studies.activity.yml'
   /user/{user}/studies/{study}/activities/{instance}/answers:
     $ref: 'endpoints/user.studies.activity.answers.yml'
+  /user/{user}/studies/{study}/activities/{instance}/uploads:
```

Comment:
I think calling this `uploads` rather than `upload-url` feels more consistent (e.g. Answers API and Uploads API) and give us more leeway to expand on it.